scripts/phase2_backfill_b.py

```
@@ -0,0 +1,574 @@
+"""
+Phase 2b: Additional critical endpoints
+"""
+
+import yaml
+from copy import deepcopy
+
+SPEC_PATH = "plex-api-spec.yaml"
+
+
+def load_spec():
+    with open(SPEC_PATH, "r") as f:
```

Ruff (0.15.15)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)